CMake/FindSnappy.cmake

```diff
@@ -0,0 +1,21 @@
+# Find the Snappy libraries
+#
+# This module defines:
+# SNAPPY_FOUND
+# SNAPPY_INCLUDE_DIR
+# SNAPPY_LIBRARY
+
+find_path(SNAPPY_INCLUDE_DIR NAMES snappy.h)
+find_library(SNAPPY_LIBRARY NAMES snappy)
+
+include(FindPackageHandleStandardArgs)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(
+    SNAPPY DEFAULT_MSG
+    SNAPPY_LIBRARY SNAPPY_INCLUDE_DIR
+)
+
+if (SNAPPY_FOUND)
+    message(STATUS "Found Snappy: ${SNAPPY_LIBRARY}")
```

Comment:
`FIND_PACKAGE_HANDLE_STANDARD_ARGS()` already prints a message (`DEFAULT_MSG`)

Reply:
Ah, cool! Did not know that. I'll rip it out then.